v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -32,8 +32,8 @@ The following table provides a list of the parameters supported by each storage
 Location                                                    | Scheme      | Host                                             | Parameters
 ------------------------------------------------------------+-------------+--------------------------------------------------+----------------------------------------------------------------------------
 Amazon                                                      | `s3`        | Bucket name                                      | [`AUTH`](cloud-storage-authentication.html#amazon-s3-specified): `implicit` or `specified` (default: `specified`). When using `specified` pass user's `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.<br><br>[`ASSUME_ROLE`](cloud-storage-authentication.html#amazon-s3-assume-role) (optional): Pass the [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the role to assume. Use in combination with `AUTH=implicit` or `specified`.<br><br>[`AWS_SESSION_TOKEN`](cloud-storage-authentication.html) (optional): For more information, see Amazon's guide on [temporary credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html). <br><br>[`S3_STORAGE_CLASS`](#amazon-s3-storage-classes) (optional): Specify the Amazon S3 storage class for created objects. Note that Glacier Flexible Retrieval and Glacier Deep Archive are not compatible with incremental backups. **Default**: `STANDARD`.
-Azure                                                       | `azure`     | Storage container                                | `AZURE_ACCOUNT_KEY`, `AZURE_ACCOUNT_NAME` <br><br>You must [url encode](https://en.wikipedia.org/wiki/Percent-encoding) your Azure account key before authenticating to Azure Storage. For more information, see [Authentication - Azure Storage](cloud-storage-authentication.html#azure-storage-authentication).
-Google Cloud                                                | `gs`        | Bucket name                                      | `AUTH`: `implicit`, or `specified` (default: `specified`); `CREDENTIALS` <br>[`ASSUME_ROLE`](cloud-storage-authentication.html#google-cloud-storage-assume-role) (optional): Pass the [service account name](https://cloud.google.com/iam/docs/understanding-service-accounts) of the service account to assume. <br><br>For more information, see [Authentication - Google Cloud Storage](cloud-storage-authentication.html#google-cloud-storage-specified).
+Azure                                                       | `azure`     | Storage container                                | `AZURE_ACCOUNT_KEY`, `AZURE_ACCOUNT_NAME`<br><br>`AZURE_ENVIRONMENT`: (optional) {% include {{ page.version.version }}/misc/azure-env-param.md %}<br><br>You must [url encode](https://en.wikipedia.org/wiki/Percent-encoding) your Azure account key before authenticating to Azure Storage. For more information, see [Authentication - Azure Storage](cloud-storage-authentication.html#azure-storage-authentication).
```

Comment:
nit: The url encode notice regarding `AZURE_ACCOUNT_NAME` now follows `AZURE_ENVIRONMENT`. What do you think about breaking this up into three bullet points, and adding the "You must" sentence to the `AZURE_ACCOUNT_NAME` entry directly? Something like:

>- AZURE_ACCOUNT_KEY: Your Azure account key. You must [url encode](https://en.wikipedia.org/wiki/Percent-encoding) your Azure account key before authenticating to Azure Storage. For more information, see [Authentication - Azure Storage](https://deploy-preview-16137--cockroachdb-docs.netlify.app/docs/v22.2/cloud-storage-authentication#azure-storage-authentication).
>- AZURE_ACCOUNT_NAME: The name of your Azure account.
>- AZURE_ENVIRONMENT: (optional) {% include {{ page.version.version }}/misc/azure-env-param.md %}

Could definitely skip the bullets to keep with the rest of the table, maybe just three separate lines one for each param?

Reply:
I hate the bullets with the difference in font, so following the formatting in the other cells. 👍

Reply:
Honestly, I think this table probably needs dismantling into sections soon.

Reply:
Making lists in tables here is always a huge challenge. I think you're probably right. I have tables like this too! 😅